nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-core/src/test/java/org/nuxeo/ecm/directory/core/CoreDirectoryInit.java

```diff
@@ -42,6 +43,20 @@
 
     public static String DOC_PWD_USER2 = "foo2";
 
+    public static String DOC_ID_USERSHA1 = "usersha1";
+
+    public static String DOC_PWD_USERSHA1 = "foosha1";
+
+    public static String ENC_PWD_USERSHA1 = ""; // computed later
+
+    public static String DOC_PWD_BADPWDSHA1 = "bad-pwd";
+
+    public static String ENC_PWD_BADPWDSHA1 = ""; // computed later
+
+    public static String DOC_PWD_NULLSHA1 = null;
+
+    public static String ENC_PWD_NULLSHA1 = ""; // computed later
+
```

Comment:
There's no reason to keep 'computed later' static fields. You should add tests in `TestPasswordHelper` in order to check that `digestWithSalt` is able to correctly handle `null` and then instantiate directly here your `ENC_*` constant with `PasswordHelper`.
But it looks like you don't need to have `ENC_*` fields as you don't use them in another place.
Please remove these fields.